buildSrc/src/main/java/Dependencies.kt

```diff
@@ -2,13 +2,26 @@ const val android_min_sdk_version = 16
 const val android_target_sdk_version = 31
 
 const val kotlin_version = "1.6.21"
+const val agp_version = "7.1.3"
```

Comment:
I think that better to be verbose to about doubts -> android_gradle_plugin_version

Reply:
Changed the name